pkg/controller/volume/persistentvolume/index_test.go

```diff
@@ -129,6 +129,15 @@ func TestMatchVolume(t *testing.T) {
 				pvc.Spec.StorageClassName = &classSilver
 			}),
 		},
+		"successful-match-vlarge": {
```

Comment:
I would also add some more test cases:
- a negative case where the requested size is larger than the PV size.
- positive case where the requested size is equal to the PV size